packages/axios/README.md

```diff
@@ -88,6 +90,39 @@ To obtain the token on the server side, call the `getAccessToken` method on the
 const accessToken = await planship.getAccessToken()
 ```
 
-## Complete SDK reference
 
-A complete reference for the `Planship` class and all related response models can be found [here](docs/classes/Planship.md).
+## Planship Customer API client class
+
+In addition to the `Planship` API client class, this library also implements the `PlanshipCustomer` client class, which provides indentical functionality, but it's initialized with a specific customer `ID` so it doesn't need to be passed to every customer specific call.
+
+```js
+import { PlanshipCustomer } from '@planship/axios'
+
+const planshipCustomer = new PlanshipCustomer(
+    'clicker-demo',                     // Your Planship product slug
+    'vader@empire.gov',                 // Customer ID
+    'https://api.planship.io',          // Planship API endpoint URL
+    '273N1SQ3GQFZ8JSFKIOK',             // Planship API client ID
+    'GDSfzPD2NEM5PEzIl1JoXFRJNZm3uAhX'  // Planship API client secret
+)
+
+// fetch a list of subscriptions
+const subscriptions = await planshipCustomer.listSubscriptions()
+
+// change the plan of the first subscription (if exists) to 'imperial'
```

Comment:
if exists -> if one exists

to 'imperial' -> to a plan with the slug 'imperial' (could also link slug to the definition of slug in our docs)